tests/Feature/JobControllerTest.php

```diff
@@ -291,16 +312,20 @@ public function testSavedJobHasCorrectTimes() : void
         $expectedCloseDate = $expectedCloseDateTime->format($dateFormat);
         $expectedCloseTime = $expectedCloseDateTime->format($timeFormat);
 
-        $newJob = $this->generateEditJobFormData();
-        $newJob['open_date'] = $openDate;
-        $newJob['close_date'] = $closeDate;
+        $job = factory(JobPoster::class)->state('draft')->create([
+            'manager_id' => $this->manager->id
+        ]);
+
+        $jobEdit = $this->generateEditJobFormData();
+        $jobEdit['open_date'] = $openDate;
+        $jobEdit['close_date'] = $closeDate;
 
         //Expected db values
-        $dbValues = array_slice($newJob, 0, 8);
+        $dbValues = array_slice($jobEdit, 0, 8);
 
         $response = $this->followingRedirects()
-            ->actingAs($this->manager->user)
-            ->post('manager/jobs/', $newJob);
+            ->actingAs($this->manager)
```

Comment:
Have to pass `$this->manager->user` to `actingAs()` here.

Reply:
Thanks, fixed